### PR TITLE
[PARTI-595] Fix `Option columns is empty` error with empty objects

### PR DESCRIPTION
**Pull request by pedro-lb**
# PR Description

This PR adds the following changes:

- Fixes the `Option columns is empty` error when passing any empty object to be ingested

## ⚠️ Dependencies

- None

### Before

image

### After

image

# How to test

## 🧪 What should be tested?

As a reviewer, you should test:

- It should not break with empty objects

## 📝 How to test?

Reproduce the following steps to test:

1. Ingest stuff w/ empty objects

### CODE REVIEW

File: src/constructor/node/buildCsvPayload/index.ts [hunk 1]
```diff
@@ -54,15 +54,23 @@ async function toCsv<T = TObject>(
   type: keyof CatalogIngestionPayloadData,
   objects: T[],
 ): Promise<ReadStream | undefined> {
+  logMemoryUsage(`Building CSV for ${type}...`);
```
Comment (guiquintelas) [inline src/constructor/node/buildCsvPayload/index.ts:57]: (Not related to this file) **Issue**: When sending only empty arrays we try to ingest an **undefined** body, leading to an invalid request - for me, it takes more than 2 min, probably timing out at some point.

Example payload:
```ts
{
  groups: [
    {},
  ],
  items: [
    {},
  ],
  variations: [
    {},
  ],
}
```

We only perform the `empty` validation before the building step.

File: src/constructor/node/buildCsvPayload/index.ts
Comment (guiquintelas) [inline src/constructor/node/buildCsvPayload/index.ts:1]: (Not related to this file) **Suggestions**: The `npm run dev` command is currently broken :/

I managed to solve this by changing the `module` and `moduleResolution` values:
image

But this change was made quite some time ago.

File: src/constructor/node/buildCsvPayload/index.ts (hunk 1 above)
Comment (guiquintelas) [reply src/constructor/node/buildCsvPayload/index.ts:57]: We should probably change the test case as well

**Review by guiquintelas — CHANGES_REQUESTED** [review]
Thanks for catching this pedro! Left some discussions

File: src/constructor/node/buildCsvPayload/index.ts
Comment (pedro-lb) [reply src/constructor/node/buildCsvPayload/index.ts:1]: Absolutely, we should! Great catch here.

Fixed in cc2e2e5bf6a8408102961c863d6a3d4776595207.

File: src/constructor/node/buildCsvPayload/index.ts (hunk 1 above)
Comment (pedro-lb) [reply src/constructor/node/buildCsvPayload/index.ts:57]: Yep we should validate this!

Added in bcc1960b429586ea24271b8406755763913999af.

**Review by guiquintelas — APPROVED** [review]
Working great!

**Pull request merged**